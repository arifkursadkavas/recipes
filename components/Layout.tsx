import * as React from 'react';

const Layout = (props) => {
  return <div className='appLayout'>{props.children}</div>;
};
export default Layout;
