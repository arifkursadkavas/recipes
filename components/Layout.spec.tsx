import * as React from 'react';
import { mount, shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Layout from './Layout';

describe('Components', () => {
  describe('Layout', () => {
    test('snapshot renders', () => {
      const component = renderer.create(<Layout />);
      let tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
