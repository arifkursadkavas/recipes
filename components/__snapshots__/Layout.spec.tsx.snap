// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Layout snapshot renders 1`] = `
<div
  className="appLayout"
/>
`;
